R/pkg/NAMESPACE

```diff
@@ -1,5 +1,9 @@
 # Imports from base R
-importFrom(methods, setGeneric, setMethod, setOldClass)
+# Do not include stats:: "rpois", "runif" - causes error at runtime
+importFrom("methods", "setGeneric", "setMethod", "setOldClass")
+importFrom("methods", "is", "new", "signature", "show")
```

Comment:
It might have to do with the R version used. I am using R 3.2.1 on my machine while this is from R 3.3.0 -- Using a later R version is obviously better.